tests/serenity/src/test/java/bla/tm/definitions/site/pantheon/UserLogInDefinition.java

```diff
@@ -24,10 +26,20 @@ public void openCreateNewAccountPage() {
     @Given("navigate to Pantheon User Account page from LogIn page")
     @When("navigate to Pantheon User Account page from LogIn page")
     public void openUserAccountPage(){
-        userLogInPage.logInToAccount();
+        userLogInPage.logInToApp(admin.username, admin.password);
+    }
+
+    @When("login to Pantheon with $username and $password")
+    public void logInToApp(String username, String password){
```

Comment:
I do not hard code login and password!!!
I get login and password data from BDD scenarios , as you see.
I created UserData class with constructor, because I need to implement negative scenarios with different data input, like data driven framework